README.md

```diff
@@ -20,41 +20,46 @@ Don't forget to use a strong, randomly generated password instead of `<PASSWORD>
 
 Create a new secret in the [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) with the following values:
 
-* `host`: Hostname of the Exasol database to connect to (this can either be a public, or a VPC local ip address)
+* `host`: VPC-local ip address of your Exasol database
 * `port`: Exasol JDBC port (default: 8563)
 * `username`: Name of an Exasol user account with `CREATE SESSION` privileges
 * `password`: Password for the account
 
+### VPC Setup
+
+The CloudWatch adapter must run in the same AWS VPC than the Exasol database. By that it can access the Exasol database using a local IP address.
```

Comment:
```suggestion
The CloudWatch adapter must run in the same AWS VPC that the Exasol database runs. By that it can access the Exasol database using a internal IP address.
```